### PR TITLE
feat: hotkey modifiers, ascend hotkey

### LINKED ISSUE #102: Feature: Ascend hotkey, hotkey modifiers
Suggested by Elzi#4864 on Discord:

image

### PR DESCRIPTION
**Pull request by nmussy**
Fixes #102

### CODE REVIEW

File: src/Hotkeys.ts [hunk 1]
```diff
@@ -33,7 +34,17 @@ document.addEventListener('keydown', event => {
...
+    let keyPrefix = '';
+    if(event.ctrlKey) {
+        keyPrefix += 'CTRL+';
+    }
+    if(event.shiftKey) {
+        keyPrefix += 'SHIFT+';
+    }
+    if(event.altKey) {
+        keyPrefix += 'ALT+';
+    }
```
Comment (KhafraDev) [inline src/Hotkeys.ts:46]:
```suggestion
    if (event.ctrlKey) {
        keyPrefix += 'CTRL+';
    } else if (event.shiftKey) {
        keyPrefix += 'SHIFT+';
    } else if (event.altKey) {
        keyPrefix += 'ALT+';
    }
```

**Review by KhafraDev — CHANGES_REQUESTED** [review]

File: src/Hotkeys.ts (hunk 1 above)
Comment (nmussy) [reply src/Hotkeys.ts:46]: I wanted to let the user combine them if they wanted, e.g. `CTRL+ALT+<key>`, in that order

Comment (KhafraDev) [reply src/Hotkeys.ts:46]: alright, then all that needs changing is the space between the if and parenthesis

Comment (nmussy) [reply src/Hotkeys.ts]: Done 👍

**Review by KhafraDev — APPROVED** [review]

**Pull request merged**